src/detect-engine.c

```diff
@@ -2940,6 +2940,12 @@ static int DetectEngineCtxLoadConf(DetectEngineCtx *de_ctx)
                          "and 255, will default to 4");
         }
     }
+    int force_applayer = 0;
+    if ((ConfGetBool("detect.force-tie-applayer-tx", &force_applayer)) == 1) {
```

Comment:
I guess we could use more strings and have an enumeration
- tie first
- tie last
- tie if only one
- tie never

And even combine with tie if content match (PACKET_ALERT_FLAG_STREAM_MATCH), tie if UDP...